apps/web/src/services/constants/applicant-details-constants.ts

```diff
@@ -0,0 +1,95 @@
+// numeric values are just placeholders for now
+
+export const professionOptionsArray = [
+  { value: 'nurse-practitioner', label: 'Nurse Practitioner' },
+  { value: 'mlt', label: 'MLT' },
+  { value: 'mrt', label: 'MRT' },
+  { value: 'midwife', label: 'Midwife' },
+  { value: 'nurse', label: 'Nurse' },
+  { value: 'occupational-therapist', label: 'Occupational Therapist' },
+  { value: 'physiotherapist', label: 'Physiotherapist' },
+  { value: 'pharmacist', label: 'Pharmacist' },
+  { value: 'sonographer', label: 'Sonographer' },
+];
+
+export const specialityOptionsArray = [
+  { value: 'chronic-disease', label: 'Chronic Disease' },
```

Comment:
I'm wondering about the pattern here. Since we have so many, it might make sense to have lower-kebab-case for the values.

Reply:
Also keeping this consistent over all the options as well is important.